src/TestSuite/IntegrationTestTrait.php

```diff
@@ -509,16 +509,17 @@ protected function _sendRequest($url, $method, $data = []): void
             }
             $this->_response = $response;
         } catch (PhpUnitError $e) {
-            throw $e;
+            $this->_exception = $e;
         } catch (DatabaseException $e) {
-            throw $e;
+            $this->_exception = $e;
         } catch (LogicException $e) {
-            throw $e;
+            $this->_exception = $e;
         } catch (Throwable $e) {
```

Comment:
`Throwable` covers all exceptions

Reply:
I think the reason here is to have different behaviors.

Reply:
Previously it was different behaviors, but now all `catch` blocks have same code, so all blocks could handled in single `catch (Throwable $e)`.